<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>trillian</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="lwLog"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblPad">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblButtons">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPlainTextEdit" name="pteMessage"/>
     </item>
     <item>
      <widget class="QPushButton" name="pbSend">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leHost">
       <property name="text">
        <string>192.168.0.135</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbPort">
       <property name="maximum">
        <number>65536</number>
       </property>
       <property name="value">
        <number>1201</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDisconnect">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
